examples/panorama/outputs.tf

```diff
@@ -7,3 +7,8 @@ output "panorama_admin_password" {
   description = "Panorama administrator's initial password."
   value       = random_password.this.result
 }
```

Comment:
Missing "sensitive"

Reply:
I'd rather not mix different changes. This PR is not for 0.15 preparation.